src/lib/button/_button-base.scss

```diff
@@ -44,15 +39,16 @@ $mat-mini-fab-padding: 8px !default;
   display: inline-block;
   white-space: nowrap;
   text-decoration: none;
-  vertical-align: baseline;
+  vertical-align: middle;
   text-align: center;
 
   // Sizing.
   margin: $mat-button-margin;
   min-width: $mat-button-min-width;
-  line-height: $mat-button-line-height;
+  height: $mat-button-height;
   padding: $mat-button-padding;
   border-radius: $mat-button-border-radius;
+  letter-spacing: 0.08929em;
```

Comment:
Letter spacing should be handled in the typography correct?

Reply:
At least at the moment we don't have letter spacing as a part of the config. I can add it in when I get to doing the typography changes.